### DISCUSSION TITLE
How can we open multiple videos as different tabs and play them one by one and close them as desired?

### DISCUSSION THREAD

**Discussion by ScarletMcLearn** [FreeTubeApp/FreeTube#4164] [Q&A] [answered]
How can we open multiple videos as different tabs and play them one by one and close them as desired?

Similar to how we can do it in browsers?

**Comment by absidue** [comment, marked as answer]
No, for the moment you can open new windows, there is a feature request open for tabs: https://github.com/FreeTubeApp/FreeTube/issues/333

**Reply by ScarletMcLearn** [reply]
Thank you. You guys are doing god's work. <3
